twiml/message/message/message-3/message-3.5.x.cs

```diff
@@ -7,8 +8,8 @@ static void Main()
     {
         var response = new MessagingResponse();
         response.Message("Store Location: 123 Easy St.",
-            action: "/SmsHandler.php", method: "POST");
+            action: new Uri("/SmsHandler.php"), method: "POST");
```

Comment:
@joliveros I think this `method: "POST"` should be `method: HttpMethod.Post`

Reply:
In the tests for twilio-csharp they just use a string. wdyt?
https://github.com/twilio/twilio-csharp/blob/master/test/Twilio.Test/TwiML/VoiceResponseTest.cs#L25

Reply:
It will work, but it's preferable to use what they provide as they may change the `method` parameter to depend on a type interface (like [here](https://github.com/twilio/twilio-csharp/blob/master/src/Twilio/Rest/Taskrouter/V1/Workspace/Worker/ReservationOptions.cs#L215)). Also, it's less prone to write typos when using the language syntax check instead of writing it manually.

Reply:
sounds like a plan. I'll update that now